### PR TITLE
prepare release 0.7

### PR DESCRIPTION
**Pull request by qbarthelemy**

### CODE REVIEW

**Comment by qbarthelemy** [comment]
@agramfort , can you share pyRiemann's Zenodo access with @sylvchev ?

File: README.md [hunk 1]
```diff
@@ -159,14 +159,22 @@ pytest
...
+  title        = {pyRiemann},
+  month        = oct,
+  year         = 2024,
+  version      = {v0.7},
   publisher    = {Zenodo},
-  version      = {v0.5},
-  doi          = {10.5281/zenodo.8059038},
-  url          = {https://doi.org/10.5281/zenodo.8059038}
+  doi          = {10.5281/zenodo.593816},
+  url          = {https://doi.org/10.5281/zenodo.593816}
```
Comment (sylvchev) [inline README.md:177]: With this url, the DOI always points towards latest version.
So we only need to add new contributors when preparing a release, and update version number+date.

Comment (qbarthelemy) [reply README.md:177]: Thx @sylvchev !

Comment (agramfort) [reply README.md:177]: done

**Pull request merged**